include/depends.mk

```diff
@@ -11,7 +11,7 @@
 
 DEP_FINDPARAMS := -x "*/.svn*" -x ".*" -x "*:*" -x "*\!*" -x "* *" -x "*\\\#*" -x "*/.*_check" -x "*/.*.swp" -x "*/.pkgdir*"
 
-find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | $(MKHASH) md5
+find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | awk -v srch="$TOPDIR" -v repl="" '{ sub(srch,repl,res); print res }' | $(MKHASH) md5
```

Comment:
Isn't it easier to use `sed` here?

Reply:
I don't trust sed as TOPDIR gets expanded and it may fail and also is problematic as the path have the same delimiter of sed (`/`)
With awk in theory we can handle all sort of strange stuff present in TOPDIR path